.github/workflows/continous-integration.yml

```diff
@@ -1148,12 +1159,45 @@ jobs:
 
       - name: Build ⚒️ Distributions
         run: poetry build
+      
+      # TODO: readd release step to puplic pypi
+      # - name: Publish to PyPI 📦
+      #    uses: pypa/gh-action-pypi-publish@717ba43cfbb0387f6ce311b169a825772f54d295
+      #    with:
+      #      user: __token__
+      #      password: ${{ secrets.PYPI_TOKEN }}
+
+      # Push to the dev registry
+      - name: Publish to internal PyPI 📦
+        uses: pypa/gh-action-pypi-publish@717ba43cfbb0387f6ce311b169a825772f54d295
+        with:
+          user: "github-ci"
+          password: ${{ secrets.PYPI_PASSWORD }}
+          repository_url: "https://pypi.rasa.com/simple/"
+          packages_dir: ${{ format('{0}/dist', github.workspace) }}
+
+      # Authenticate and push to the release registry
+      - id: "auth-release"
+        name: Authenticate with gcloud for release registry 🎫
+        uses: "google-github-actions/auth@ceee102ec2387dd9e844e01b530ccd4ec87ce955"
+        with:
+          credentials_json: "${{ secrets.RASA_OSS_RELEASE_ACCOUNT_KEY }}"
+
+      - name: "Set up Cloud SDK"
+        uses: "google-github-actions/setup-gcloud@62d4898025f6041e16b1068643bfc5a696863587"
 
-      - name: Publish to PyPI 📦
-        uses: pypa/gh-action-pypi-publish@c7f29f7adef1a245bd91520e94867e5c6eedddcc
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
```

Comment:
@tmbo could you pin this to a SHA please

Reply:
this is just for debugging and will be removed before merge :)